nuget/Microsoft.Windows.CsWinRT.nuspec

```diff
@@ -22,9 +22,10 @@
     <file src="$cswinrt_exe$"/>
     <file src="Microsoft.Windows.CsWinRT.props" target="build"/>
     <file src="Microsoft.Windows.CsWinRT.targets" target="build"/>
-    <file src="Microsoft.Windows.CsWinRT.Prerelease*.targets" target="build"/>
     <file src="Microsoft.Windows.CsWinRT.Authoring.targets" target="build"/>
-    <file src="Microsoft.Windows.CsWinRT.Authoring.targets" target="buildTransitive"/>
+    <file src="Microsoft.Windows.CsWinRT.Prerelease*.targets" target="build"/>
+    <file src="Microsoft.Windows.CsWinRT.Authoring.Transitive.targets" target="build"/>
+    <file src="Microsoft.Windows.CsWinRT.Authoring.Transitive.targets" target="buildTransitive"/>
```

Comment:
This is almost certainly not what's intended. The fact that the target doesn't have the package name means this file is gonna be ignored.

Reply:
@jlaanstra We have targets work in Authoring.targets that renames the file when packaging a custom component, so it doesn't get ignored. 

```
<!-- From the target "CsWinRTIncludeHostDlls" -->
<Content Include="$(CsWinRTPath)buildTransitive\Microsoft.Windows.CsWinRT.Authoring.Transitive.targets" 
               Pack="true" 
               PackagePath="buildTransitive\$(AssemblyName).targets;build\$(AssemblyName).targets" />
```